apps/ui/src/views/Space/Editor.vue

```diff
@@ -344,7 +356,9 @@ watch(
   async id => {
     if (id) return true;
 
-    const newId = await createDraft(spaceKey.value);
+    const newId = await createDraft(spaceKey.value, {
+      privacy: props.space.privacy === 'shutter' ? 'shutter' : 'none'
+    });
```

Comment:
We have different places that call this method (two in this file, others when creating proposal from treasury) and in other cases this flag won't be set.